dvc/repo/experiments/__init__.py

```diff
@@ -701,6 +699,7 @@ def _collect_executor(self, rev, executor, stages, result, lock):
         finally:
             lock.release()
 
+    @no_type_check
```

Comment:
I was not sure of specific types in the below args/kwargs type. They have `Iterable` type but not sure of what. For now, disable type-checking in that function.